assets/sass/authorize-application/_googlesitekit-authorize-application.scss

```diff
@@ -141,6 +141,10 @@
 	color: $c-authorize-application-sys-light-on-surface;
 	font-family: $f-primary;
 
+	a {
+		color: $c-authorize-application-vivid-blue;
+	}
```

Comment:
Instead of applying the anchor tag color to `#wpcontent`, please apply it to the `#wpbody-content` section as specified in the IB. This change is necessary because `#wpcontent` includes the navigation bar, which this style should not affect.

### WP Content
![Screenshot 2024-05-28 at 2 35 25 PM](https://github.com/google/site-kit-wp/assets/24408261/38b758ef-966a-44b9-abd8-c05796955889)

---

### WP Body Content 
![Screenshot 2024-05-28 at 2 35 37 PM](https://github.com/google/site-kit-wp/assets/24408261/1c283d07-c193-495a-a993-a84b934ada07)